File: src/main/java/top/zbeboy/isy/service/data/ScoreTypeService.java
package top.zbeboy.isy.service.data;

import top.zbeboy.isy.domain.tables.pojos.ScoreType;

import java.util.List;

/**
 * Created by lenovo on 2017-07-23.
 */
public interface ScoreTypeService {

    /**
     * 查询全部类型
     *
     * @return 全部
     */
    List<ScoreType> findAll();
}

File: src/main/java/top/zbeboy/isy/service/data/ScoreTypeService.kt
package top.zbeboy.isy.service.data

import top.zbeboy.isy.domain.tables.pojos.ScoreType

/**
 * Created by zbeboy 2017-12-12 .
 **/
interface ScoreTypeService {
    /**
     * 查询全部类型
     *
     * @return 全部
     */
    fun findAll(): List<ScoreType>
}